### DISCUSSION TITLE
Error: "It seems that you are not enrolled in this course yet. Please enrol first"

### DISCUSSION THREAD

**Discussion by m-waqas88** [Cyfrin/security-and-auditing-full-course-s23#7] [General]
I tried opening an issue but I was not able to find any specific category for such issue so I simply started this discussion. I hope that I am abiding the guidelines.

When I click on "Next lesson", the lecture is changed but a toast error appears saying "It seems that you are not enrolled in this course yet. Please enrol first"
![Screenshot 2023-12-05 at 5 00 45 PM](https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/62002637/885d6a02-53cf-4e34-aec0-22db8a3f33b4)

Furthermore, when the lecture is completed the checkbox is not checked. Even when I click on that checkbox nothing happens.
![Screenshot 2023-12-05 at 5 08 08 PM](https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/62002637/24dbbcaa-82ee-4ab3-90aa-b78a4a69add6)

**Comment by m-waqas88** [comment]
@PatrickAlphaC

**Reply by PatrickAlphaC** [reply]
Thanks for this! Have you signed up?

**Reply by m-waqas88** [reply]
@PatrickAlphaC Yes I had. I signed up with my email "m.waqas171288@gmail.com", but I did not receive access code.

I am also enrolled in Johnny Time's Smart Contract Hacking Course, I got the access code from that community, so I simply entered that access code.

I think in your database, there is no access code against my email address but I actually have the access code that is why I got the access to the course content.

**Reply by cromewar** [reply]
Hello @m-waqas88 in order to provide you with personalized support can you send us an email to vasiliy@cyfrin.io with the code you used? so we can check?

**Reply by m-waqas88** [reply]
@cromewar access code is sent via email.

**Reply by m-waqas88** [reply]
@cromewar @PatrickAlphaC any update on this please?

**Reply by drained69** [reply]
To solve this, you will need to go to the home page and click on the course. That is how to enroll.

**Reply by m-waqas88** [reply]
@Chinwuba22 yup it worked. Thanks man.
